### ISSUE TITLE
How to use it for superglue

### ISSUE THREAD

**Issue by ChrisLLLLL** [ducha-aiki/pydegensac#12]
Thanks for the excellent work! I want to use it for superglue. But superglue will not get <DMatch>. In other words, i don't know how to get the "tentatives"(draw_matches(kps1, kps2, tentatives, img1, img2, H, mask)). Could you help me answer my doubts?
Looking forward to your reply.

**Comment by ducha-aiki** [comment]
Is it a question about pydegensac or draw_matches?

**Comment by ChrisLLLLL** [comment]
yes!
i want to use pydegensac for superglue.

**Comment by ducha-aiki** [comment]
SuperGlue returns you matching xy in image0 and xy in image1. They are your src_pts and dst_pts, which you feed into pydegensac, which gives you inliers and F matrix.

**Comment by ChrisLLLLL** [comment]
then the"draw_matches(kps1, kps2, tentatives, img1, img2, H, mask)" the tentatives how to get? it format is DMatch, superglue has not. and DMatch  seems to be only available through "cv2.BFMatcher" ?

**Comment by ducha-aiki** [comment]
Then it is not a question about pydegensac ;)

You can install kornia and kornia_moons, and then draw like this:


```python3
 import kornia.feature as KF
 import torch
 from kornia_moons.feature import *
 laf1 = KF.laf_from_center_scale_ori(torch.from_numpy(src_pts).reshape(1,-1,2),
                                            torch.ones(len(src_pts)).reshape(1,-1,1,1),
                                            torch.zeros(len(src_pts)).reshape(1,-1,1))
 laf2 = KF.laf_from_center_scale_ori(torch.from_numpy(dst_pts).reshape(1,-1,2),
                                                torch.ones(len(src_pts)).reshape(1,-1,1,1),
                                                torch.zeros(len(src_pts)).reshape(1,-1,1))
 draw_LAF_matches(laf1,laf2, torch.stack([torch.arange(len(src_pts)), torch.arange(len(src_pts))], dim=-1),
img1, img2, inliers, 
                  draw_dict = {'inlier_color': (0.1, 0.8, 0.1), 'vertical': True})    
```

**Comment by ChrisLLLLL** [comment]
ok, thank you ! i will try it
